src/main/java/io/cdap/plugin/google/common/GoogleAuthBaseConfig.java

```diff
@@ -162,13 +173,10 @@ public ValidationResult validate(FailureCollector collector) {
         try {
           GoogleDriveClient client = new GoogleDriveClient(this);
 
-          // validate auth
-          validateCredentials(collector, client);
```

Comment:
so this check is not redundant, IIRC when google drive API is not enabled in the project using service account authentication.

Can you please verify these cases and attach the screenshot in the description after removal of this check?

- Google Drive API is not enabled in the project
- Google Drive API is enabled but does not have access on the folder/file.
- Google Drive API is enabled but folder/file does not exist.

Reply:
All three scenerios are tested and snapshot attached in the description, In case of no access or directory not exist, api returns same result and same is shown on the UI. In case of drive apis not enabled, it will throw different exception as given in snapshot in description